### ISSUE TITLE
Multiple naming sequences in integration_tests

### ISSUE THREAD

**Issue by Shaikh-Ubaid** [lfortran/lfortran#1576]
It seems we have multiple naming sequences for `integration_tests` corresponding to arrays.

For example, one sequence starts here
https://github.com/lfortran/lfortran/blob/6eabb9d4f6f0b211e24356011ba19516844083aa/integration_tests/CMakeLists.txt#L242

Another is here
https://github.com/lfortran/lfortran/blob/6eabb9d4f6f0b211e24356011ba19516844083aa/integration_tests/CMakeLists.txt#L766-L767

A similar is seen for integration_tests corresponding to `array_op`:
https://github.com/lfortran/lfortran/blob/6eabb9d4f6f0b211e24356011ba19516844083aa/integration_tests/CMakeLists.txt#L258-L270

Another `array_op` sequence:
https://github.com/lfortran/lfortran/blob/6eabb9d4f6f0b211e24356011ba19516844083aa/integration_tests/CMakeLists.txt#L778-L781

I think we should ideally maintain/follow a unique sequence corresponding for each feature (for example, a sequence for `arrays` and another sequence for `array_op`). The names of these sequences seem to differ by singularity (`array_01` and `arrays_01`). If possible, I think we should merge these to one sequence.

**Comment by certik** [comment]
Yes, we should follow uniform naming scheme, using `_01`, `_02`, etc. suffix, which allows the tests to be naturally sorted using "ls".
